editor/editor.go

```diff
@@ -10,9 +11,12 @@ const defaultEditor = "nano"
 
 // Cmd returns a *exec.Cmd editing the given path with $EDITOR or nano if no
 // $EDITOR is set.
-func Cmd(path string) *exec.Cmd {
+func Cmd(path string) (*exec.Cmd, error) {
+	if os.Getenv("SNAP_REVISION") != "" {
+		return nil, fmt.Errorf("Did you install with Snap? Glow is sandboxed and unable to open an editor. Please install Glow with Go or another package manager to enable editing.")
```

Comment:
@caarlos0 it's so long, but that's the clearest and most concise I could think of 😂